### PR TITLE
Update Server Availability.workbook

### PR DESCRIPTION
**Pull request by WKahnZA**
adding fix for duplicate RG names due to case being different eg. ResourceGroup and resourcegroup are the same but come up as two different RGs

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Virtual Machines/Availability/Server Availability.workbook [hunk 1]
```diff
@@ -274,5 +265,8 @@
       "name": "query - 0"
     }
   ],
+  "fallbackResourceIds": [
```
Comment (gardnerjr) [inline Workbooks/Virtual Machines/Availability/Server Availability.workbook:268]: you don't need this `fallbackResourceIds` section either, surprising it didn't warn in the validation build.  no harm here specifically but surprising nontheless.

File: Workbooks/Virtual Machines/Availability/Server Availability.workbook [hunk 2]
```diff
@@ -182,22 +181,17 @@
               "columnMatch": "$gen_group",
               "formatter": 13,
               "formatOptions": {
-                "linkTarget": "Resource",
+                "linkTarget": null,
```
Comment (gardnerjr) [inline Workbooks/Virtual Machines/Availability/Server Availability.workbook:184]: did changing this fix something or was it unintentional?  do individual computer rows still show up as links?

File: Workbooks/Virtual Machines/Availability/Server Availability.workbook (hunk 1 above)
Comment (WKahnZA) [reply Workbooks/Virtual Machines/Availability/Server Availability.workbook:268]: seems to be there every time I copy from the portal, i'll remove it

File: Workbooks/Virtual Machines/Availability/Server Availability.workbook (hunk 2 above)
Comment (WKahnZA) [reply Workbooks/Virtual Machines/Availability/Server Availability.workbook:184]: unintentional, that did break the link good catch. fixed and resubmitting

**Review by gardnerjr — APPROVED** [review]
FYI: now that i look closer at the query changes, you could have just done `extend ResourceGroup=tolower(ResourceGroup) ` and not had to change any of the other column names/settings/etc

**Pull request merged**